apps/docs/app/global.css

```
@@ -2,31 +2,769 @@
+}
+
+a:focus-visible,
+button:focus-visible,
+input:focus-visible {
+  outline: 2px solid var(--sync-cyan);
+  outline-offset: 3px;
+}
+
 #nd-docs-layout {
   --fd-layout-width: 100vw;
+  background: var(--night-ink);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)